example/websocket/server/sync/websocket_server_sync.cpp

```diff
@@ -108,9 +108,9 @@ int main(int argc, char* argv[])
             acceptor.accept(socket);
 
             // Launch the session, transferring ownership of the socket
-            std::thread{std::bind(
-                &do_session,
-                std::move(socket))}.detach();
+            std::thread{
+                &do_session, std::move(socket)
+            }.detach();
```

Comment:
Style:
```
std::thread(
    &do_session,
    std::move(socket)).detach();
```

Reply:
I copied the curly braces from the existing code :/

Reply:
Yeah the curly braces were dumb